dotcom-rendering/src/components/StickyBottomBanner.importable.tsx

```diff
@@ -59,14 +60,12 @@ type RRBannerConfig = {
 };
 
 const getBannerLastClosedAt = (key: string): string | undefined => {
-	// eslint-disable-next-line no-restricted-syntax -- FIXME-libs-storage
-	const item = localStorage.getItem(`gu.prefs.${key}`) as undefined | string;
+	const item = storage.local.get(`gu.prefs.${key}`) as undefined | string;
```

Comment:
I don’t think we need to do the casting anymore, because we’re checking the it’s a string below

```suggestion
	const item = storage.local.get(`gu.prefs.${key}`);
```